core/internal/settings/settings.go

```diff
@@ -112,6 +112,11 @@ func (s *Settings) GetStartTime() time.Time {
 	return time.UnixMicro(int64(seconds * 1e6))
 }
 
+// The root directory for the current working directory.
```

Comment:
this doesn't make sense to me

what's an example of what this can be? is it like "./wandb"? or does it have to do with the user's scripts?

Reply:
oh it is indeed poorly phrased by default it would be the abs path of the cwd, and `wandb` dir would be the root of that path. So something like:
`wandb-dir = <root-dir>/wandb`